### PR TITLE
student

### PR DESCRIPTION
**Pull request by Chiral13**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -234,3 +240,9 @@
   github: Beta-King
   image: Beta_KIng.jpg
   twitter: Yash_Beta_King
+  
+  - name: Chirag kumar
+  country: India
+  github: INFERNOMK1
+  image: ChiragKumar.jpg
+  twitter: INFERNOMK1
```
Comment (nikit19) [inline _data/students.yml:248]: Please remove this

**Review by nikit19 — CHANGES_REQUESTED** [review]
Looks good but you have added your details twice.

File: _data/students.yml (hunk 1 above)
Comment (Chiral13) [reply _data/students.yml:248]: ok

Comment (Chiral13) [reply _data/students.yml:248]: can you check the new one?

Comment (abishekvashok) [reply _data/students.yml:248]: @INFERNOMK1 don't add yourself twice

**Comment by abishekvashok** [comment]
Please close PRs that you don't need. Also please don't open multiple PRs for the same cause, make changes in that PR itself

**Pull request closed without merging**